bootstrap_cfn/fab_tasks.py

```diff
@@ -478,6 +491,12 @@ def _validate_fabric_env():
     elif not os.path.isfile(env.config):
         sys.exit("\n[ERROR] Config file %s does not exist" % str(env.config))
 
+    # keyname is mandatory in cfn_create, optional in others.
+    # if _validate_fabric_env() is called by cfn_create, then check keyname.
+    # otherwise not.
+    if called_by_cfn_create and env.keyname is None:
+            sys.exit("\n[ERROR] Please specify a keyname, e.g 'keyname:keyops'")
+
```

Comment:
I can appreciate why it's a good idea to mandate it, but, this is a breaking change. A reasonable compromise might be to accept that some stacks have it defined in their config, and warn rather than error. Also, deleting the keyname from the template-deploy example?